lib/grape-swagger/endpoint.rb

```diff
@@ -102,7 +102,9 @@ def path_item(routes, options)
     end
 
     def method_object(route, options, path)
+      puts "lib/grape-swagger/endpoint.rb:105 DEBUG OUTPUT FOR GRAPE-SWAGGER: #{route.options}"
```

Comment:
Good catch, I totally forgot to remove that when I cherry-picked over the original commit